### DISCUSSION TITLE
Migrating existing nodes to new panels

### DISCUSSION THREAD

**Discussion by KiwiTheMemer** [pelican-dev/panel#109] [Ideas]
### Discussed in https://github.com/pelican-dev/panel/discussions/82

<div type='discussions-op-text'>

<sup>Originally posted by **KiwiTheMemer** April 14, 2024</sup>
Make a way to import an existing node into a different panel. This could provide benefit in a lot of ways and in my opinion was a big weakness with pterodactyl.</div>

**Comment by OkayThereBud** [comment]
I agree

**Comment by MennoGamed** [comment]
This would certainly help, especially for people with more than 1 node.

**Comment by crocofied** [comment]
agreed
